tests/test_io.py

```diff
@@ -280,10 +286,10 @@ def mock_urlretrieve(
 
 
 @pytest.mark.parametrize(
-    "compress, font_size, use_styler, width, height",
+    ("compress", "font_size", "use_styler", "width", "height"),
     [
         # Default font size, no compression, DataFrame
-        (False, 14, False, 457, 875),
+        (False, 14, False, 458, 875),
```

Comment:
This is a bit unexpected, I don't think I changed any functional part, but this width changed from `457` to `458`